engine/apps/api/views/escalation_chain.py

```diff
@@ -120,14 +121,19 @@ def perform_update(self, serializer):
     @action(methods=["post"], detail=True)
     def copy(self, request, pk):
         name = request.data.get("name")
+        team_id = request.data.get("team")
         if name is None:
             raise BadRequest(detail={"name": ["This field may not be null."]})
         else:
             if EscalationChain.objects.filter(organization=request.auth.organization, name=name).exists():
                 raise BadRequest(detail={"name": ["Escalation chain with this name already exists."]})
 
         obj = self.get_object()
-        copy = obj.make_copy(name)
+        try:
+            team = request.user.available_teams.get(public_primary_key=team_id)
```

Comment:
Looks like this will break when copying an escalation chain without a team, right? I'd recommend adding a unit tests to check this scenario.

Reply:
Should be fixed in https://github.com/grafana/oncall/pull/1658/commits/0fedcd614bb662aa66a1eccfa478c88ef2687c89